### PR TITLE
Update Paint Timing pages

### PR DESCRIPTION
**Pull request by Elchi3**
### Description

This PR updates https://developer.mozilla.org/en-US/docs/Web/API/PerformancePaintTiming and redirects https://developer.mozilla.org/en-US/docs/Web/API/Paint_Timing_API to it.

### Motivation

https://github.com/openwebdocs/project/issues/62

### Additional details

The Paint_Timing_API overview page doesn't bring any additional value. We [discussed](https://github.com/openwebdocs/project/issues/62#issuecomment-1303278689) that we want to present the Performance API as a single API and not fragment it per the individual specifications anymore. A Performance API guide will present all of these things together. 

### Related issues and pull requests

See also https://github.com/mdn/content/pull/22031 where the groups were reduced to a single "Performance API" group already.

### CODE REVIEW

**Comment by bsmth** [comment]
Thanks, Florian, taking a look 👀

File: files/en-us/glossary/first_paint/index.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ tags:
   - Web Performance
 ---
 
-**First Paint**, part of the [Paint Timing API](/en-US/docs/Web/API/Paint_Timing_API), is the time between navigation and when the browser renders the first pixels to the screen, rendering anything that is visually different from the default [background color](/en-US/docs/Web/CSS/background-color) of the [body](/en-US/docs/Web/API/Document/body). It answers the question "Is it happening?"
+**First Paint**, is the time between navigation and when the browser renders the first pixels to the screen, rendering anything that is visually different from the default [background color](/en-US/docs/Web/CSS/background-color) of the [body](/en-US/docs/Web/API/Document/body). It answers the question "Is it happening?"
```
Comment (bsmth) [inline files/en-us/glossary/first_paint/index.md:10]: I think "Is it happening" comes from [this web.dev page](https://web.dev/user-centric-performance-metrics/#defining-metrics). It only makes sense in the context of the other questions, so I would reword this to something like the following:

```suggestion
**First Paint** is the time between navigation and when the browser first renders pixels to the screen, rendering anything that is visually different from the default [background color](/en-US/docs/Web/CSS/background-color) of the [body](/en-US/docs/Web/API/Document/body). It is the first key moment in page load times and will answer the question "Has the server started to render the page?"
```

Maybe it can be improved further based on [the terminology](https://www.w3.org/TR/paint-timing/#sec-terminology).

File: files/en-us/web/api/performancepainttiming/index.md [hunk 2]
```diff
@@ -2,28 +2,31 @@
...
+There are two key paint moments this API provides:
+
+- {{Glossary("First paint")}} (FP): Time when anything is rendered. Note that the marking of the first paint is optional, not all user agents report it.
+- {{Glossary("First contentful paint")}} (FCP): Time when the first bit of DOM text or image content is rendered.
+
+A third key paint moment is provided by the {{domxref("LargestContentfulPaint")}} API:
+
+- {{Glossary("Largest contentful paint")}} (LCP): Render time of the largest image or text block visible within the viewport, recorded from when the page first begins to load.
+
+The data this API provides helps you minimize the time that users have to wait before they can see the site's content start to appear. You can make the site feel more engaging and responsive (better performing) for your users.
```
Comment (bsmth) [inline files/en-us/web/api/performancepainttiming/index.md:21]: Minor suggestion:

```suggestion
The data this API provides helps you minimize the time that users have to wait before they can see the site's content start to appear. Decreasing the time until these key paint moments make sites feel more responsive, performant, and engaging for your users.
```

**Review by bsmth — APPROVED** [review]
A couple of comments / minor suggestions, but otherwise looks good, tnx 👍🏻

File: files/en-us/web/api/performancepainttiming/index.md [hunk 3]
```diff
@@ -2,28 +2,31 @@
...
 slug: Web/API/PerformancePaintTiming
 page-type: web-api-interface
-tags:
-  - API
-  - Interface
-  - Paint Timing
-  - Performance Timeline API
-  - PerformancePaintTiming
-  - Reference
-  - Web Performance
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply files/en-us/web/api/performancepainttiming/index.md:12]: No reason really. I think we want to remove them anyways and I was playing with removal to see if it had any effect. I can add it back if needed.

File: files/en-us/web/api/performancepainttiming/index.md [hunk 4]
```diff
@@ -2,28 +2,31 @@
...
 
-An application can register a {{domxref("PerformanceObserver")}} for "`paint`" {{domxref("PerformanceEntry","performance entry types")}} and the observer can retrieve the times that paint events occur. Use this information to help identify areas that take too long to provide a good user experience.
+There are two key paint moments this API provides:
+
+- {{Glossary("First paint")}} (FP): Time when anything is rendered. Note that the marking of the first paint is optional, not all user agents report it.
+- {{Glossary("First contentful paint")}} (FCP): Time when the first bit of DOM text or image content is rendered.
+
+A third key paint moment is provided by the {{domxref("LargestContentfulPaint")}} API:
+
+- {{Glossary("Largest contentful paint")}} (LCP): Render time of the largest image or text block visible within the viewport, recorded from when the page first begins to load.
```
[the remark this answers is not in the document]
Comment (Elchi3) [reply files/en-us/web/api/performancepainttiming/index.md:19]: Yes, sorry should have mentioned that my next PR will deal with the LCP pages.

File: files/en-us/web/api/performancepainttiming/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (bsmth) [reply files/en-us/web/api/performancepainttiming/index.md:12]: I didn't know they're not used for anything, so I guess there's no reason to keep them 👍🏻
